gtsfm/evaluation/plot_metrics.py

```diff
@@ -30,21 +36,33 @@ def create_metrics_plots_html(json_path: str, output_dir: str) -> None:
         "rotation_averaging_metrics.json",
         "translation_averaging_metrics.json",
         "data_association_metrics.json",
-        "bundle_adjustment_metrics.json"
+        "bundle_adjustment_metrics.json",
     ]
+
+    if Path(colmap_files_dirpath).exists():
```

Comment:
what is the behavior of Path() if colmap_files_dirpath is None?

Reply:
The behavior was TypeError. Your new design avoids that though.